File: dest/8_for.swift
/**
 * For loop iterates through anything that provides an iterator.
 * See http://kotlinlang.org/docs/reference/control-flow.html#for-loops
 */
func main(args: [String]) {
    for arg in args {
        print(arg)
    }

    // or
    print("")
    for i in args.indices {
        print(args[i])
    }
}

main([])

File: test-src/8_for.kt
/**
 * For loop iterates through anything that provides an iterator.
 * See http://kotlinlang.org/docs/reference/control-flow.html#for-loops
 */
fun main(args: Array<String>) {
    for (arg in args) {
        println(arg)
    }

    // or
    println()
    for (i in args.indices) {
        println(args[i])
    }
}